python/GafferDispatch/SystemCommand.py

```diff
@@ -81,6 +84,9 @@ def execute( self ) :
 		for name, value in environmentVariables.items() :
 			env[name] = str( value )
 
-		subprocess.check_call( command, shell = True, env = env )
+		if self["shell"].getValue() == False :
+			command = shlex.split( command )
+
+		subprocess.check_call( command, shell = self["shell"].getValue(), env = env )
```

Comment:
It's probably worth avoiding the second `getValue()` call here - evaluate inputs once to avoid unnecessary overhead.